projects/VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -1896,6 +1948,28 @@ public void Get_BirthRecord_Roundtrip()
             DeathRecord dr2 = ije1.ToRecord();
             Assert.Equal("242123", dr2.BirthRecordId);
         }
+
+          [Fact]
+        public void Get_BirthRecordChild_Roundtrip()
+        {
+            DeathRecord dr = DeathRecord1_JSON;
+            Assert.Equal("717171", dr.BirthRecordChildId);
+            IJEMortality ije1 = new IJEMortality(dr);
+//            Assert.Equal("242123", ije1.BCNO);
```

Comment:
should this be removed or changed to "717171"?

Reply:
Removed.   BCNO != BirthRecordChildId...  IJE has no equivalent to BirthRecordChildId.